src/components/TodoFilter/TodoFilter.tsx

```diff
@@ -1,30 +1,58 @@
-export const TodoFilter = () => (
-  <form className="field has-addons">
-    <p className="control">
-      <span className="select">
-        <select data-cy="statusSelect">
-          <option value="all">All</option>
-          <option value="active">Active</option>
-          <option value="completed">Completed</option>
-        </select>
-      </span>
-    </p>
+import { memo } from 'react';
 
-    <p className="control is-expanded has-icons-left has-icons-right">
-      <input
-        data-cy="searchInput"
-        type="text"
-        className="input"
-        placeholder="Search..."
-      />
-      <span className="icon is-left">
-        <i className="fas fa-magnifying-glass" />
-      </span>
+type Props = {
+  sortedBy: string;
+  query: string;
+  setSortedBy: (newSotedBy: string) => void;
```

Comment:
There is a typo in the parameter name `newSotedBy`. It should be `newSortedBy` to maintain consistency and avoid confusion.